A compile error occurred in src/pages/_app.tsx and src/pages/index.tsx

**Describe the bug**
①error - ./src/pages/_app.tsx:1:0
Module not found: Can't resolve '../styles/globals.scss'

②error - ./src/pages/index.tsx:4:0
Module not found: Can't resolve '../styles/Home.module.css'

**To Reproduce**
Steps to reproduce the behavior:
①compile ./src/pages/_app.tsx file
②compile ./src/pages/index.tsx:4:0

--- a/src/pages/_app.tsx
+++ b/src/pages/_app.tsx
@@ -1,4 +1,4 @@
-import '../styles/globals.css'
+import '../styles/globals.scss'
 import type { AppProps } from 'next/app'
 
 function MyApp({ Component, pageProps }: AppProps): JSX.Element {

